src/libraries/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSourceActivity.cs

```diff
@@ -568,4 +573,9 @@ public abstract class ObservableInstrument<T> : Instrument where T : struct
         protected ObservableInstrument(Meter meter, string name, string? unit, string? description, System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, object?>> tags) : base(meter, name, unit, description) { throw null; }
         protected abstract System.Collections.Generic.IEnumerable<Measurement<T>> Observe();
     }
+    public sealed class HistogramAdvice<T> where T : struct
+    {
```

Comment:
We are trying to list the types in this file alphabetically. Could you please move this to its right place above?